tests/mappers/test_action_mapper.py

```diff
@@ -96,7 +96,7 @@ def test_parse_job_xml_and_configuration_minimal_green_path(
     </configuration>
 </action>
 """
-        action_node = cast(ET.Element, ET.fromstring(action_node_str))
+        action_node: Element = cast(ET.Element, ET.fromstring(action_node_str))
```

Comment:
```suggestion
        action_node = cast(ET.Element, ET.fromstring(action_node_str))
```
This is not needed because the cast method comes from the typings package. The second things - I think it's better to write an ET.Element. It will be more consistent. _elementtree is a private package.

Reply:
+1 for the ET.Element, I will change it.
I wanted to introduce this to make it clearer for the reader what type that object is.